jupyter/Dockerfile

```diff
@@ -1,24 +1,50 @@
 FROM ubuntu:18.04
 
-RUN apt-get update || true
-RUN apt-get install -y openjdk-11-jdk-headless
-RUN apt-get install -y python3-pip git
-RUN pip3 install jupyter
-RUN apt-get update \
-    && DEBIAN_FRONTEND=noninteractive apt-get install -y locales \
-    && sed -i -e 's/# en_US.UTF-8 UTF-8/en_US.UTF-8 UTF-8/' /etc/locale.gen \
-    && dpkg-reconfigure --frontend=noninteractive locales \
-    && update-locale LANG=en_US.UTF-8
-RUN apt-get install -y curl
+ARG NB_USER="sagemaker-user"
```

Comment:
SageMaker does not force user to create this user. User can always manually configure the entry point if necessary.    Take a look at [here](https://github.com/aws-samples/sagemaker-studio-custom-image-samples/tree/main/examples/scala-image)